src/graphql.rs

```diff
@@ -241,9 +241,9 @@ impl Revision {
         let commit = transaction.repo().find_commit(self.commit_id)?;
 
         let warnings = filter::compute_warnings(&transaction, self.filter, commit.tree()?)
-            .iter()
-            .map(|warn| Warning {
-                text: warn.to_string(),
+            .into_iter()
```

Comment:
Out of curiosity, why is it better?

Reply:
It avoids cloning the strings. (`iter` yields borrowed values that have to be (essentially) cloned via `to_string`, whereas `into_iter` allows for cheap moves.)

Reply:
thanks!